.github/workflows/build-and-test.yml

```diff
@@ -60,7 +60,7 @@ jobs:
       if: github.event_name == 'pull_request'
       run: |
         echo "GIT_SLUG=${{github.event.pull_request.head.repo.full_name}}" >> $GITHUB_ENV
-        echo "GIT_BRANCH=${{ env.GITHUB_HEAD_REF_SLUG }}" >> $GITHUB_ENV
+        echo "GIT_BRANCH=${{ env.GITHUB_HEAD_REF_SLUG_CS }}" >> $GITHUB_ENV
```

Comment:
I find it strange that the default (without `_CS`) is basically wrong

Reply:
The previous branch I have created had `_` on it as well and the slug was converting `_` to `-`. Does this flag fix this issue as well?

Reply:
Ah I missed that it had an underscore, will fix that also.

Reply:
OK I have removed the slug action altogether since I found the built in Github equivalent. I've tested it with both upper case and underscores in a PR on my fork [here](https://github.com/Bensuo/sycl-blas/actions/runs/679855280) that's running now, if you check the PR environment vars step again you can see the correct branch name is passed.